app/database/queries/breeze_buddy/user_management.py

```
@@ -0,0 +1,436 @@
+                    DELETE FROM users
+                    WHERE role = 'merchant' AND merchant_ids::jsonb ? $1
+                    RETURNING id
+                """
+                deleted_merchant = await conn.fetchrow(delete_merchant_query, merchant_id)
+                
+                if not deleted_merchant:
+                    return False, 0
+                
+                logger.info(f"Deleted merchant {merchant_id} and {deleted_users_count} users")
+                return True, deleted_users_count
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)